tests/rules/test_quoted_strings.py

```diff
@@ -14,6 +14,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from tests.common import RuleTestCase
+from yaml.reader import ReaderError as yaml_reader_ReaderError
```

Comment:
We never do such imports renaming: can you use simply `ReaderError`? Or `yaml.reader.ReaderError`?